docs/src/user/tasks.rst

```diff
@@ -18,6 +18,111 @@ Predictions are generated by *Model Developers* when they define :meth:`rafiki.m
+    │  ├── 0015023cc06b5362d332b3.json
+    │  ├── 001b4a31684c8fc6e2cfbb70304354978317c429.json
+    │  ...
+    ...
+    │
+    └──metadata.csv                                          # if additional information is provided for above JSON files, user can add a metadata.csv
+
+- JSON file includes ``body_text``, providing list of paragraphs in full body which can be used for question answering. ``body_text`` can contain different entries, only the "text" field of each entry will be read.
+
+The essential structure of JSON file is:
+
+.. code-block:: text
+
+    {
+        "body_text": [                      # list of paragraphs in full body
+            {                               
+                "text": <str>,              # text body for first entry, this is must-have 
+            },
+            ...                             # other entries, paragraph dicts look the same as above
+        ],
+    }
+
+
+- ``metadata.csv`` is not strictly required. User can provide additional information with it, i.e. authors, title, journal and publish_time, mapping to JSON files by sha values. ``cord_uid`` serves unique values serve as the entry identity. Time related entry, like ``publish_time``, is suggested to be in Date format, otherwise, General format is recommended.
+
+An example of ``metadata.csv`` entry: 
+    =====================       =====================
+    Column Names                Column Values 
+    ---------------------       --------------------- 
+    cord_uid                    zjufx4fo                
+    sha                         b2897e1277f56641193a6db73825f707eed3e4c9  
+    source_x                    PMC                   
+    title                       Sequence requirements for RNA strand transfer during nidovirus ... 
+    doi                         10.1093/emboj/20.24.7220         
+    pmcid                       PMC125340                
+    pubmed_id                   11742998                
+    license                     unk                   
+    abstract                    Nidovirus subgenomic mRNAs contain a leader sequence derived ...
+    publish_time                2001-12-17             
+    =====================       =====================
+    
+
+Query Format 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+
+    - The pretrained model should be fine-tuned with a dataset first to adapt to particular question domains when necessary. 
+    - Otherwise, following the question, input should contain relevant information (context paragraph or candidate answers, or both), whether or not addresses the question. 
+
+Query is in JSON format. It could be a <str list> of a single question in ``questions`` field. While the relevant information is provided in query, the question always comes first, followed by additional information. We use “\n” separators between different parts of the input. Model will only read the ``questions`` field. 
```

Comment:
`different parts` refer to `question` and `context` fields. I suggest we do not separate them since we do not want to make particular formats for every types of datasets